controllers/operands/kubevirt.go

```diff
@@ -294,7 +301,73 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 	return kvObject
 }
 
-func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfiguration, error) {
+func getRatesFromConfigMap(configMap *corev1.ConfigMap) (*kubevirtcorev1.ReloadableComponentConfiguration, error) {
+
+	qps, ok := configMap.Data[qps]
```

Comment:
`qps` is already used in this file as a constant. please change the name of one of them (e.g. make the constant more readable).

Reply:
Done.